### PR TITLE
fix : Session details have correct alignment

### LINKED ISSUE #1930: Wrong indentation of text in Session detail activity
**Actual Behaviour**

The text shown in each line is wrongly indented  (The track text and below four lines).

**Expected Behaviour**

The text shown in each line should be correctly indented.

**Steps to reproduce it**

We need to change the way the text is represented by changing layout orientations.

**LogCat for the issue**

Provide logs for the crash here

**Screenshots of the issue**

screenshot_20170925-144937

**Would you like to work on the issue?**

Yes

**Target Milestone to solve the issue**

Till which [milestone](https://github.com/fossasia/open-event-android/milestones) will you be fixing the issue?

### PR DESCRIPTION
**Pull request by dreadpool2**
Fixes #1930

Changes: The session details have been aligned correctly.

Screenshots for the change: 
screenshot_20171022-041334

### CODE REVIEW

**Comment by dreadpool2** [comment]
@heysadboy @iamareebjamal please review this

**Review by scandindian — CHANGES_REQUESTED** [review]
I still see the colon is not having proper spacing, maybe we could do it like this,

Track ______ : 
Location___ : 

etc.

**Comment by dreadpool2** [comment]
@heysadboy Actually the whole part is a defined string, should I modify the defined strings then?

**Comment by dreadpool2** [comment]
@heysadboy Is this alignment correct ?
screenshot_20171023-194230

**Comment by scandindian** [comment]
@dreadpool2 Maybe we could put "Start" and "Time" on the same line. @iamareebjamal Please see.

**Comment by iamareebjamal** [comment]
Yes, please change it to duration

9:00 AM - 9:10 AM

**Comment by dreadpool2** [comment]
@iamareebjamal Okay!

**Comment by dreadpool2** [comment]
The session detail activity now looks like this
screenshot_20171024-165336

**Comment by dreadpool2** [comment]
@the-dagger ,@iamareebjamal please review this

File: android/app/src/main/res/layout/content_session_detail.xml [hunk 1]
```diff
@@ -33,133 +33,167 @@
                     android:layout_height="wrap_content"
                     android:layout_marginBottom="@dimen/layout_margin_medium"
                     android:textColor="@color/primary_text" />
-
-                <RelativeLayout
+                <LinearLayout
```
Comment (the-dagger) [inline android/app/src/main/res/layout/content_session_detail.xml]: I'll suggest using a GridLayout here instead.
Will reduce the XML nesting a lot.

**Review by the-dagger — CHANGES_REQUESTED** [review]

File: android/app/src/main/res/layout/content_session_detail.xml (hunk 1 above)
Comment (dreadpool2) [reply android/app/src/main/res/layout/content_session_detail.xml]: @the-dagger okay!

**Comment by dreadpool2** [comment]
@the-dagger done the changes!

**Comment by the-dagger** [comment]
@dreadpool2 Can you add a screenshot as well?

**Comment by dreadpool2** [comment]
@the-dagger done

**Comment by dreadpool2** [comment]
@the-dagger done

**Review by the-dagger — APPROVED** [review]

**Pull request merged**